content/posts/2020-03-22-rmarkdown-current-date.es.Rmarkdown

```diff
@@ -0,0 +1,72 @@
+---
+title: Mostrar la fecha actual automáticamente en un informe Rmarkdown
+author: Thomas Neitmann
+date: '2020-03-22'
+slug: rmarkdown-current-date
```

Comment:
Could you translate this as well. This slug will be used as the URL of the page.